docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -24,6 +25,189 @@
+		* New permissible value: `Not Applicable`
+* Altered `demographic` Entity
+	* New property: `country_of_birth`
+	* New property: `education_level`
+	* New property: `marital_status`
+* Altered `diagnosis` Entity
+	* New deprecated property: `metastasis_at_diagnosis_site`
+	* New property: `clark_level`
+	* New property: `contiguous_organ_invaded`
+	* New property: `ensat_clinical_m`
+	* New property: `ensat_pathologic_n`
+	* New property: `ensat_pathologic_stage`
+	* New property: `ensat_pathologic_t`
+	* New property: `first_symptom_longest_duration`
+	* New property: `measurement_type`
+	* New property: `melanoma_known_primary`
+	* New property: `tumor_burden`
+	* New property: `ulceration_indicator`
+	* Changes made to `sites_of_involvement`
+		* Changes made to `items`
+			* New permissible value: `Ascites`
+			* New permissible value: `Common or Superficial Femoral Vein`
+			* New permissible value: `Distant Nodes`
+			* New permissible value: `Distant Organ`
+			* New permissible value: `Gastrointestinal Tract`
+			* New permissible value: `Great Blood Vessel`
+			* New permissible value: `Groin`
+			* New permissible value: `Head, Face or Neck, NOS`
+			* New permissible value: `Inferior Vena Cava`
+			* New permissible value: `Lymph Node, Axillary`
+			* New permissible value: `Lymph Node, Inguinal`
+			* New permissible value: `Lymph Node, Regional`
+			* New permissible value: `Lymph Node, Subcarinal`
+			* New permissible value: `Mediastinum`
+			* New permissible value: `Pelvic Vein, Common, External or Iliac`
+			* New permissible value: `Peritoneal Cavity`
+			* New permissible value: `Renal Vein`
+			* New permissible value: `Scalp`
+			* New permissible value: `Spinal Cord`
+			* New permissible value: `Urethra`
+			* New permissible value: `Vertebral Canal`
+			* New permissible value: `Vulva, NOS`
+* Altered `exposure` Entity
+	* New property: `exposure_duration_hrs_per_day`
+	* Changes made to `exposure_type`
+		* New permissible value: `Dust, NOS`
+* Altered `family_history` Entity
+	* New property: `relative_deceased`
+	* New property: `relative_smoker`
+* Altered `follow_up` Entity
+	* New property: `discontiguous_lesion_count`
+	* Changes made to `comorbidities`
+		* Changes made to `items`
+			* New permissible value: `Pneumonia, NOS`
+	* Changes made to `first_event`
+		* New permissible value: `Presented with Metastases`
+	* Changes made to `risk_factors`
+		* Changes made to `items`
+			* New permissible value: `Pneumonia, NOS`
+	* Changes made to `timepoint_category`
+		* New permissible value: `Within 3 Months of Surgery`
+* Altered `masked_methylation_array` Entity
+	* Changes made to `platform`
+		* New permissible value: `Illumina Methylation Epic v2`
+* Altered `methylation_beta_value` Entity
+	* Changes made to `platform`
+		* New permissible value: `Illumina Methylation Epic v2`
+* Altered `molecular_test` Entity
+	* New property: `staining_intensity_scale`
+	* Changes made to `gene_symbol`
```

Comment:
Collapse second_gene_symbol into this one.

Reply:
Lumped second_gene_symbol changes with gene_symbol changes